### ISSUE TITLE
Add publisher/journal: Canadian Science Publishing

### ISSUE THREAD

**Issue by mariaguerreiro** [CDL-Dryad/dryad-product-roadmap#3187]
**Name to appear on membership page:** 
Canadian Science Publishing

**Payment plan:**
Annual membership + Quarter DCPs

**Journals included:** 
FACETS
Arctic Science
Drone Systems and Applications
Canadian Journal of Fisheries and Aquatic Sciences

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
https://docs.google.com/spreadsheets/d/1QyDVBWSR2OrivpKK_fbTtVr7_Klq5Rm2zRuoTGTbAtg/edit?usp=sharing

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
https://docs.google.com/spreadsheets/d/1QyDVBWSR2OrivpKK_fbTtVr7_Klq5Rm2zRuoTGTbAtg/edit?usp=sharing

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
Waiting for confirmation - will add to this ticket in the coming days.

**Email contacts to discuss integration of journal metadata with Dryad:**
judy.hum-delaney@cdnsciencepub.com

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [ ] start integration process (track on new ticket)

**Comment by jleighherzog** [comment]
All journals should be configured to automatically be placed in PPR status upon submission:

- FACETS
- Arctic Science
- Drone Systems and Applications
- Canadian Journal of Fisheries and Aquatic Sciences

CSP will send through metadata that includes 'rejected' and 'withdrawn' decision terms. Please configure our site to auto-withdraw the related datasets that have these decision terms attached.

**Comment by jleighherzog** [comment]
@ryscher : As discussed, CSP has sent through several test emails for their email integration: 

I can confirm that I've received test emails for each submission listed below: 
cjzr-2024-0001 - Submitted --> Accepted
cjzr-2024-0002 - Submitted --> Rejected
cjzr-2024-0003 - Submitted --> Accepted
cjzr-2024-0004 - Submitted --> Rejected
cjzr-2024-0006 - The first was a "submitted" notification, the second an "accepted" notification, and the third a "rejected" notification

Can you please check the functionality?
